Miscellaneous/HackerRank/MatrixScript.py

```diff
@@ -0,0 +1,19 @@
+first_multiple_input = input().rstrip().split()
+
+n = int(first_multiple_input[0])
+
+m = int(first_multiple_input[1])
+
+matrix = []
+
```

Comment:
You may put the logic in a function and only have the input and output function in the main code. Also add proper comments to explain your approach.

Reply:
Add appropriate comments and then this PR will be good to merge.